src/mon/MDSMonitor.cc

```diff
@@ -1165,6 +1165,23 @@ bool MDSMonitor::preprocess_command(MonOpRequestRef op)
       }
     }
     r = 0;
+  } else if (prefix == "fs feature ls") {
```

Comment:
Hmm reading the list of features out of the code of the monitor which receives the command is a bit weird; should we forward it to the leader? I guess upgrade cases aren't real key here but for stuff that goes into monitor commands I'm used to either forwarding or having the agreed list of options .

Reply:
updated